helpers.js

```diff
@@ -1,8 +1,39 @@
 // fetch data
 const API_URL = "https://gist.githubusercontent.com/maratgaip/44060c688fcf5f2b7b3985a6d15fdb1d/raw/e93c3dce0826d08c8c6e779cb5e6d9512c8fdced/restaurant-menu.json";
 
-export const getData = async () => {
-    const response = await fetch(API_URL);
-    const data = await response.json();
-    return data;
-}
\ No newline at end of file
+// export const getData = async () => {
+//     const response = await fetch(API_URL);
+//     const data = await response.json();
+//     return data;
+// }
+
+// export const getData = fetch(API_URL)
+//     .then(response => response.json())
+//     .then(data => data)
+
+export const data = [
```

Comment:
Let's use API from the googleDoc that was provided by Seytech.
We don't need to save those data here.

Reply:
I don't have these in my local, I used them to test but then before pushing I deleted all those and fixed them.  Not sure how they appeared there LOL